src/config.py

```diff
@@ -0,0 +1,12 @@
+DEVICE = 'cpu'
+LANGUAGE = 'en'
+MAX_CHAR_LEN = 140
+MODEL_ID = 'v3_en'
+SAMPLE_RATE = 24000
+# SPEAKER_LIST = ['en_0', 'en_29', 'en_41', 'en_110']
```

Comment:
Your config should probably contain any file paths you need to use anywhere in your code as well. Never hardcode file paths since they can easily change in a reorg.

Reply:
Commented lines removed.